packages/loader/src/GLTFLoader.ts

```diff
@@ -19,14 +19,18 @@ export class GLTFLoader extends Loader<GLTFResource> {
    * @remarks If use loader after releasing, we should release again.
    */
   static release(): void {
-    MeshoptDecoder.release();
+    getMeshoptDecoder().then((meshoptDecoder) => {
+      meshoptDecoder.release();
+    });
   }
 
   override initialize(_: Engine, configuration: EngineConfiguration): Promise<void> {
     const meshOptOptions = configuration.glTF?.meshOpt;
     if (meshOptOptions) {
-      MeshoptDecoder.workerCount = meshOptOptions.workerCount;
-      MeshoptDecoder.useWorkers();
+      return getMeshoptDecoder().then((meshoptDecoder) => {
+        meshoptDecoder.workerCount = meshOptOptions.workerCount;
+        meshoptDecoder.useWorkers();
```

Comment:
The calling method is not concise enough. Why not make `workerCount` an optional parameter of `workerCount` to enhance readability and reduce code?

Reply:
Done.